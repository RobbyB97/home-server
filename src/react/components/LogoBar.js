import React from 'react'

const LogoBar = () => (
    <div id="logo-bar">
        <ul>
            <li>
                <a href="https://www.github.com/robbyb97" target="_blank">
                    <img src="/assets/images/icons/github.svg" />
                    <p>
                        GitHub
                    </p>
                </a>
            </li>
            <li>
                <a href="https://www.linkedin.com/in/robbyb97" target="_blank">
                    <img src="/assets/images/icons/linkedin.png" />
                    <p>
                        LinkedIn
                    </p>
                </a>
            </li>
        </ul>
    </div>
)

export default LogoBar