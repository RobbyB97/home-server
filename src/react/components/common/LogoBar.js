import React from 'react'

const LogoBar = () => (
    <div id="logo-bar">
        <ul>
            <li>
                <a href="https://www.github.com/robbyb97" target="_blank">
                    <img src="/dist/images/icons/github.svg" />
                </a>
            </li>
            
            <li>
                <a href="https://www.linkedin.com/in/robbyb97" target="_blank">
                    <img src="/dist/images/icons/linkedin.png" />
                </a>
            </li>
        </ul>
    </div>
)

export default LogoBar