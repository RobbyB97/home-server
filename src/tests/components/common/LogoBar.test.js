import React from 'react'
import {shallow} from 'enzyme'

import LogoBar from '../../../react/components/common/LogoBar'


test('Render LogoBar', () => {
    const wrapper = shallow(<LogoBar />)

    expect(wrapper).toMatchSnapshot()
})