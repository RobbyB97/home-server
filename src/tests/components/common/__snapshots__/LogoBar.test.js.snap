// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render LogoBar 1`] = `
<div
  id="logo-bar"
>
  <ul>
    <li>
      <a
        href="https://www.github.com/robbyb97"
        target="_blank"
      >
        <img
          src="/dist/images/icons/github.svg"
        />
      </a>
    </li>
    <li>
      <a
        href="https://www.linkedin.com/in/robbyb97"
        target="_blank"
      >
        <img
          src="/dist/images/icons/linkedin.png"
        />
      </a>
    </li>
  </ul>
</div>
`;
